vaadin-components-gwt/src/main/java/com/vaadin/components/grid/GridComponent.java

```diff
@@ -288,14 +288,18 @@ public IndexBasedSelectionModel getSelectionModel() {
         return (IndexBasedSelectionModel) grid.getSelectionModel();
     }
 
-    public void refresh() {
+    public void forceRefresh() {
```

Comment:
I would've wanted to have the original `refresh()` method and an overload `refresh(boolean)` but unfortunately GWT compile doesn't apparently allow overloads, so I had to rename the original method to `forceRefresh()`. It can be inlined to replace all the references with `refresh(true)` but since it's about matter of taste I didn't want to do that before I hear some opinions from you guys :smile: